src/style.css

```diff
@@ -0,0 +1,19 @@
+.blocks {
+  margin: 0;
+}
+
+.box {
+  height: 100px;
+}
+
+.redBlock {
```

Comment:
пиши назви класів через `-`
```suggestion
.red-block {
```

Reply:
исправил